lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -138,7 +138,8 @@ const mapStateToProps = (state) => {
           ? '#' + route.color
           : route?.color || '#5A5A5A'
       // Try to populate this attribute, which is required for the vehicle popup to appear.
-      vehicle.routeShortName = vehicle.routeShortName || route?.shortName
+      vehicle.routeShortName =
+        vehicle.routeShortName || route?.shortName || true
```

Comment:
Why add `true` as opposed to something like "unknown"?

Reply:
In case one day the overlay did decide to render the string, I didn't want it to possibly even have an unlocalized string to deal wtih